content-repo/extra-docs/releases/20.11.1.md

```diff
@@ -0,0 +1,1116 @@
+##### Palo Alto Networks AutoFocus v2
+Fixed an issue where the **autofocus-sample-analysis** command would fail when some data is 'Not Available'.
+
+---
+
+### Azure Feed Pack v1.0.3
+#### Integrations
+##### Azure Feed
+- Added Azure service in the Indicators field.
+- Updated the Docker image to the latest version.
+
+---
+
+### Base Pack v1.3.40
+#### Scripts
+##### CommonServerPython
+- Removed log print of the object stored in the integration cache.
+- Added support to **IgnoreAutoExtract** in the **CommandResults** object.
+- Added a default value for the *url_suffix* argument in the **BaseClient** _http_request method.
+- Maintenance and stability enhancements.
+
+##### GetIncidentsByQuery
+Added support for wildcards in the incident types argument.
+
+##### DBotMLFetchData
+Updated support for custom phishing types in the incident fetch query.
+
+##### DBotTrainTextClassifierV2
+Updated calculation of *minimum precision* to be the minimum score of the precision per-class scores.
+
+##### DBotPreProcessTextData
+Removed log messages for texts that exceed the maximum allowed text length.
+
+##### GetMLModelEvaluation
+Merged the two result entries to a single entry.
+
+##### DBotBuildPhishingClassifier
+Added an enhancement to fetch only incidents of labels which are relevant for the model training.
+
+---
+
+### Bastille Networks Pack v1.0.2 (Partner Supported)
+#### Integrations
+##### Bastille Networks
+- Documentation and metadata improvements.
+- Upgraded the Docker image to demisto/python3:3.8.6.12176.
+
+---
+
+### Bmc Helix Remedyforce Pack v1.0.2
+#### Integrations
+##### BMC Helix Remedyforce
+- Added support for the *impact_id* argument in the **bmc-remedy-incident-create** and the **bmc-remedy-incident-update** commands.
+- Upgraded the Docker image to demisto/python3:3.8.6.13358.
+
+---
+
+
+### CVE Search Pack v1.0.3
+#### Integrations
+##### CVE Search v2
+Updated the commands, **cve** and **cve-latest**, to return a different result for each indicator.
+
+#### Scripts
+##### cveReputation
+Fixed an issue where the script returned DBotScore 3 when no CVE was found.
+
+---
+
+### Carbon Black Enterprise Live Response Pack v1.1.0
+#### Integrations
+##### VMware Carbon Black EDR (Live Response API)
+- The ***cb-file-get*** command is not longer deprecated and is available for use.
+- Added the *download* argument to the ***cb-get-file-from-endpoint*** command. If "true" will download the file from the CarbonBlack server. Default is "true". Set to false for large files.
+
+#### Playbooks
+##### New: Carbon Black Live Response - Download File
+Download a file from a sensor.
+
+##### New: Carbon Black Live Response - Wait Until Command Complete
+Polling command status until finished with an error or complete.
+
+##### New: Carbon Black Live Response - Create active session
+Create an active session. If the active session already exists, will use the existing session.
+
+---
+
+### Chronicle Pack v1.1.3 (Partner Supported)
```

Comment:
Need to fix formatting. It has extra bullets:

![image](https://user-images.githubusercontent.com/1395797/100005856-46d9e800-2dd2-11eb-826c-9f25075ce226.png)